src/main/java/org/myrobotlab/service/Servo.java

```diff
@@ -292,25 +296,39 @@ public ServiceConfig apply(ServiceConfig c) {
     return c;
   }
 
+  @Override
+  public ServiceConfig getFilteredConfig() {
+    ServoConfig sc = (ServoConfig) super.getFilteredConfig();
+    Set<Listener> removeList = new HashSet<>();
+    if (sc.listeners != null) {
+      for (Listener listener : sc.listeners) {
+        if (listener.callback.equals("onServoEnable") || listener.callback.equals("onServoDisable") || listener.callback.equals("onEncoderData")

```

Comment:
Why are we filtering these methods out here?

Reply:
The servo pub subs more which is good, but it makes the config bloated .... not sure the best way to handle it ... gut feeling is to remove the filtering,
But if you start going down that path, you should treat it all the same.  The webgui is by far the biggest user of pub/sub subscriptions .. and the files get very big.  

Since there is code in both the WebGui & Servo and other services (e.g. like attachServo() ) which automatically (in code) create the subscription ... they "can" and currently are filtered out, because they are automatically setup again when attached.

Filtering is more an asthetic thing ... trying to be kinder gentler to people, when the attach gives you 300 lines of subscriptions.

@AutonomicPerfectionist and I talked about this, and didn't resolve on a perfect solution...

Perhaps its containing the pub/sub mapping in different files, but there are pros/cons with that as well, not to mention should perhaps be done in a separate pr.